src/main.cpp

```diff
@@ -291,6 +358,85 @@ int main(int argc, char **argv){
         if (visualize)
             visualization::visualize(world, robots, collision, config.show_full_map.value(), bbox, robot_radius);
 
+        visualization_msgs::Marker doors, moving_objects, undefined_objects;
+        doors.header.frame_id = "origin";
```

Comment:
This should be `map`. Even if you keep the double map frame thing. The frame used by the rest of the simulator and end-user, probably the one in the middle of the map, should be named `map`.